src/withTranslation.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+
+export default function withTranslation(WrappedComponent: React.ComponentType<any>) {
+  return function WithTranslation(props: any) {
+    return (
+      <I18nextProvider i18n={window.i18next} defaultNS="@openmrs/openmrs-form-engine-lib">
```

Comment:
We need to be careful here: it is appropriate for `@openmrs/openmrs-form-engine-lib` to be used as the namespace for strings defined in the _form engine library_, but _form specific translations_ (like concepts and labels) should be confined to a form-specific namespace.

Reply:
Hi @ibacher!
I understand that this is to be done not to conflict the common translation keys between the forms, but everytime we load a form, if we override the form's translations in the previously presented translations, I believe we can skip having and handling namespaces based on forms. Hence saving the effort of handling translations for 2 kinds: 1. Static text translations and 2. Form translations.

What do you think?

Reply:
@vasharma05 I'm ok with not having a namespace-per-individual-form, but I do think we need to distinguish between the namespace for translating things inside the form-engine-lib and the namespace for translating form-specific stuff, if only because the former only needs to be loaded once, no?

Reply:
@ibacher , I will not be adding the static translations again, I will just be adding the form translations in the existing and loaded resource. I referred this from Ngx Form entry.